app.rb

```diff
@@ -1,9 +1,42 @@
+require 'rack/handler'
 require 'sinatra/base'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
```

Comment:
Before submitting code for review, it's good practice to remove any test code, commented out code and debugging stuff.